setup.py

```diff
@@ -3,8 +3,9 @@
 
 from setuptools import setup
 from glob import glob
+import os
 
-
+setup_dir = path.abspath(os.path.dirname(__file__))
```

Comment:
The test is failing because you need to put `setup_dir = os.path.abspath(os.path.dirname(__file__))`